server/service/service.go

```diff
@@ -77,8 +78,14 @@ func New(conf *config.Config) (*Service, error) {
 
 func generateDataEncryptionKeyFile(dataEncryptionKeyFile string, err error) ([]byte, error) {
 	var key []byte
-	if tos.FileExists(dataEncryptionKeyFile) {
-		file, err := tos.ReadFile(dataEncryptionKeyFile)
+
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Very nice.
One thing, since this code is duplicate, it would be better to declare it only once.
(Maybe like below?)
```go
fileFullPath := filepath.Join(userHomeDir, dataEncryptionKeyFile)
```

Reply:
Thanks. It is applied in [c72b322](https://github.com/medibloc/panacea-data-market-validator/pull/76/commits/c72b3223a751cdd42c7902ec9502230667d24a2f)